.github/workflows/ci.yml

```diff
@@ -19,9 +36,14 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - name: install, build, and test
       run: |
+        npm install -g npm
         npm install
+        npm install --no-save @fortawesome/fontawesome-svg-core@${{ matrix.fontawesome-svg-core }} vue@${{ matrix.vue }}
```

Comment:
@jasonlundien Looks like we are missing the installation of `free-solid-svg-icons`?